packages/base/src/scripts/org/forgerock/commons/ui/common/components/hoc/withRouter.js

```diff
@@ -60,7 +62,7 @@ define([
                         })
                     };
 
-                return React.createElement(WrappedComponent, _.extend({}, this.props, { router: router }));
+                return React.createElement(WrappedComponent, _.extend({}, this.props, { router }));
```

Comment:
Does this go through transpilation? Is this acceptable by r.js?

Reply:
I don't think goes through transpilation, you're right, I'll revert it.